backend/match4everyone/settings/common.py

```diff
@@ -117,7 +117,12 @@
 SITE_ID = 1
 X_FRAME_OPTIONS = "SAMEORIGIN"
 CMS_TEMPLATES = [
+    ("home-without_content.html", "Empty page template"),
```

Comment:
should this be an underscore?

Reply:
I named it like this but it can have any name, not sure what you mean